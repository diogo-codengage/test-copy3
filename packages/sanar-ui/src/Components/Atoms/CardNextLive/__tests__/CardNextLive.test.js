import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESCardNextLive from '../'

it('renders correctly', () => {
    const component = (
        <ESCardNextLive
            title='Live de Correção da prova SUS-SP 2019'
            date='27/04/2019 às 10h'
        />
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
