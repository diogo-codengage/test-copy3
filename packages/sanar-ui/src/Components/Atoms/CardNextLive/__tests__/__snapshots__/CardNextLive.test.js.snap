// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ant-card es-card ant-card-bordered ant-card-small"
>
  <div
    className="ant-card-body"
    style={Object {}}
  >
    <h4
      aria-label={null}
      className="ant-typography es-card-next-live__title"
      style={
        Object {
          "WebkitLineClamp": null,
        }
      }
    >
      Live de Correção da prova SUS-SP 2019
    </h4>
    <span
      aria-label={null}
      className="ant-typography es-card-next-live__date ant-typography-disabled"
      style={
        Object {
          "WebkitLineClamp": null,
        }
      }
    >
      27/04/2019 às 10h
    </span>
  </div>
</div>
`;
